ajax/libs/mnist/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "mnist",
+  "filename": "mnist.js",
+  "description": "The goal of this library is to provide an easy-to-use way for training and testing MNIST digits for neural networks (either in the browser or node.js).",
+  "version": "1.1.0",
+  "author": {
+    "name": "Juan Cazala",
+    "email": "cazala@mail.sfsu.edu",
+    "url": "http://juancazala.com/"
+  },
+  "keywords": [
+    "mnist",
+    "digits"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cazala/mnist.git"
+  },
+  "npmName": "mnist",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "",
+      "files": [
+        "src/**/*"
```

Comment:
Maybe in this case, we can just add the flie under `dist`, as this example show https://github.com/cazala/mnist/blob/master/visualizer.html#L6.
Because we don't host json files, also the minified file is built by [`browserify`](https://github.com/cazala/mnist/blob/master/package.json#L7). Not sure the files under `src` can be used in web front-end.

Would you please let me know that the files under `src` can be used in web front-end?
Many thanks

Reply:
@sufuf3 You are right. The files in src folder cannot be used in web. Removed